server.js

```diff
@@ -40,7 +40,7 @@ const params = {
 };
 
 const endpoints = [
-  { table: 'action_log',        all: '/api/v1/actions',           one: '/api/v1/actions/:id',           reqParams: params.log,         getSecurity: true,  updateSecurity: false   },
+  { table: 'action_log',        all: '/api/v1/actions',           one: '/api/v1/actions/:id',           reqParams: params.log,         getSecurity: false,  updateSecurity: false   },
```

Comment:
don't we want getting all actions from the action log to be secured? maybe a specific get to get all actions from the action_log pertaining to a specific user instead?